pages/faqs.md

```diff
@@ -236,3 +236,14 @@ That’s up to you, Solid just gives you the choice.
 ## Is it wise to trust people to make ethical decisions on such a complicated issue?
 There is a lot of variation in what people feel to be right and wrong, and ultimately it is down to you to decide what to do with your data. Solid does try to help you make informed choices by providing relevant information and by constantly validating the accuracy of that information.
 
+## Are apps vulnerable to “Origin” being escaped? 
+
+The Solid specification warns that the “trusted apps” feature is new and experimental because 
```

Comment:
```suggestion
The Solid specification [warns](https://github.com/solid/web-access-control-spec#adding-trusted-web-apps) that the “trusted apps” feature is new and experimental because 
```